enterprise/frontend/src/embedding-sdk/components/public/CreateQuestion/CreateQuestion.tsx

```diff
@@ -4,6 +4,7 @@ import type { InteractiveQuestionProps } from "../InteractiveQuestion";
 
 type CreateQuestionProps = Omit<InteractiveQuestionProps, "questionId">;
 
+/** @deprecated Use `InteractiveQuestion` with `isSaveEnabled={true}` and without specifying `questionId` instead. */
```

Comment:
Should we be more aggressive? I'm not sure all IDEs even show a warning for `@deprecated`
Maybe rendering something in the UI?